region/maridia/inner-pink/Colosseum.json

```diff
@@ -1194,6 +1194,283 @@
+              "requires": [
+                "canTrickyDashJump",
+                "canInsaneJump",
+                "canGravityJump",
+                "canMomentumConservingTurnaround"
+              ],
+              "note": [
+                "This applies to Draygon's Room.",
+                "Start about a tile away from the wall, gain run speed, and time a pause to unequip Gravity after Samus jumps;",
+                "hold down and back through the unpause to buffer a turnaround."
+              ],
+              "detailNote": [
+                "Gain extra run speed of $4.0 or $4.1.",
+                "A last-frame jump with a speed of $4.1 gives a 5-frame window for the pause/turnaround,",
+                "while a second-to-last-frame jump with a speed of $4.0 gives a 2-frame window."
+              ],
+              "devNote": [
+                "If Spring Ball is available, the frame window for the turnaround with $4.0 speed can be larger,",
+                "but it doesn't seem like a big enough difference to justify separate logical requirements."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 41.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canInsaneJump",
+                "canMomentumConservingTurnaround"
+              ],
+              "note": [
+                "This applies to Blue Brinstar Energy Tank Room (Power Bomb blocks broken), Bowling Alley (middle), and Basement."
+              ],
+              "detailNote": [
+                "With ideal starting position, this has a 2-frame window for the jump,",
+                "and a 1-frame or 3-frame window for the turnaround based on the jump timing",
+                "(with a last-frame jump giving the larger window for the turnaround).",
+                "A ceiling mockball is also possible but more difficult, requiring a last-frame jump and a 2-frame window for the morph."
+              ],
+              "devNote": [
+                "The trick is more difficult if using the full runway in the long rooms Bowling Alley or Basement,",
+                "requiring a last-frame jump and a 2-frame window for the turnaround."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 39.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 2]],
+              "requires": [
+                "canMomentumConservingTurnaround"
+              ],
+              "note": ["This applies to Metal Pirates Room."]
+            },            
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 39.4375,
+              "speedBooster": true,
+              "obstructions": [[5, 2]],
+              "requires": [
+                "canInsaneJump",
```

Comment:
They're both expert to me.  The morph is kinda harder but is also more natural to do once I understood it.